[OSDOCS-20645]: CQA: Importing a hosted cluster (#115145)
openshift/openshift-docs: modules/hcp-import-manual.adoc

[id="hcp-import-manual_{context}"]
= Manually importing hosted clusters

[role="_abstract"]

> Typically, hosted clusters are automatically imported to {mce-short} after the hosted control plane becomes available. However, if needed, you can import hosted clusters manually.

Reviewer: ```suggestion
Typically, hosted clusters are automatically imported to {mce-short} after the hosted control plane becomes available. However, you can import hosted clusters manually as required.
```
